components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/SCIMCommonConstants.java

```diff
@@ -83,6 +83,10 @@
 
     public static final String READ_ONLY_USER_CLAIM = "http://wso2.org/claims/identity/isReadOnlyUser";
 
+    public static final String PROVISIONED_SOURCE_CLAIM = "http://wso2.org/claims/identity/userSource";
```

Comment:
`http://wso2.org/claims/identity/userSource` claim couldn't be used for this improvement. This claim is already used to keep the user store of the user in Asgardeo.

Reply:
A new claim is expected to introduce and will update the PR with that claim URI before merging this.